docker/server-slim/src/main/docker/image-bootstrap.properties

```diff
@@ -1,7 +1,9 @@
-Configuration.rootFile=grpc-api-docker.prop
+Configuration.rootFile=grpc-api.prop
 io.deephaven.configuration.PropertyInputStreamLoader.override=io.deephaven.configuration.PropertyInputStreamLoaderTraditional
 LoggerFactory.silenceOnProcessEnvironment=true
 workspace=.
 devroot=.
 stdout.toLogBuffer=true
 stderr.toLogBuffer=true
+
+deephaven.console.type=groovy
```

Comment:
deliberately setting this to unconditionally be groovy?

Reply:
In `deephaven/server-slim` we don't have python installed; so setting it unconditionally here. I think a better solution would be for the software layer to choose the best value when it's unset (right now it defaults to python), but there are some unclean configuration implementation details that I'd rather not bake in to the software layer w/o some cleaner refactoring.

Reply:
ah i misunderstood, looking locally I see it makes sense.